src/Okta.Sdk/IOktaClient.cs

```diff
@@ -16,6 +14,25 @@ namespace Okta.Sdk
     /// </summary>
     public interface IOktaClient
     {
+        /// <summary>
+        /// Gets the request context.
+        /// </summary>
+        RequestContext RequestContext { get; }
```

Comment:
Do we need to expose this?

Reply:
It seemed useful to do so; I'll see if not doing so is feasible.

Reply:
Removed RequestContext property from IOktaClient.